doc/pachctl/pachctl.rst

```diff
@@ -1,51 +0,0 @@
-Pachctl Command Line Tool
```

Comment:
I think there was a reason we needed both an .rst and an .md file for pachctl, but I can't figure it out right now. Let me try to figure out why that was the case again before approving this